AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -367,6 +425,16 @@ - (void)startWithChannelGroup:(id<MSChannelGroupProtocol>)channelGroup
   [super startWithChannelGroup:channelGroup appSecret:appSecret transmissionTargetToken:token fromApplication:fromApplication];
   [self.channelGroup addDelegate:self];
   [self processLogBufferAfterCrash];
+  if (self.isEnabled) {
```

Comment:
Can you extract this as a method?
`[self processMemoryWarningInLastSession]` should be more readable in the method. 😄